front/components/layout/Links.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+  <span class="links d-flex flex-row align-center justify-space-between">
+    <nuxt-link
+      class="link"
+      :class="{ active: page === 'explore' }"
+      :to="{ name: 'ideas-all' }"
+    >
+      <img
+        v-if="page === 'explore'"
+        class="active-bulb"
+        src="~/assets/images/navbar/bulb-active.svg"
+      />
+      <img v-else src="~/assets/images/navbar/bulb.svg" />
+      <span class="link-text">
+        <span class="text-shadow"></span>
+        <span class="text">Explore</span>
+      </span>
+    </nuxt-link>
+    <nuxt-link
+      class="link"
+      :class="{ active: page === 'post' }"
+      :to="{ name: 'ideas-create' }"
+    >
+      <img
+        v-if="page === 'post'"
+        src="~/assets/images/navbar/post-active.svg"
```

Comment:
post and post-active same svg. just different fill and stroke. SO this should be one component with fill and stoke param and used for both img

Reply:
Okay, I tried a few different ways of doing this but I settled on creating components for each SVG icon. This solved two problems.

1. The SVG icons no longer flash on mobile when switching screens
2. The fill and stroke colors are set instead of loading a new SVG image